pom.xml

```diff
@@ -1,14 +1,59 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+<project
+    xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
   <modelVersion>4.0.0</modelVersion>
+
   <groupId>io.dapr</groupId>
-  <artifactId>client</artifactId>
-  <packaging>jar</packaging>
-  <version>0.1.0-preview01</version>
-  <name>dapr-client</name>
+  <artifactId>client-parent</artifactId>
+  <packaging>pom</packaging>
+  <version>0.1.0-preview02</version>
+  <name>dapr-client-parent</name>
   <description>gRPC client for Dapr.</description>
   <url>https://dapr.io</url>
 
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <grpc.version>1.25.0</grpc.version>
+    <protobuf.version>3.11.0</protobuf.version>
+    <protoc.version>3.10.0</protoc.version>
+    <os-maven-plugin.version>1.6.2</os-maven-plugin.version>
+    <maven-dependency-plugin.version>3.1.1</maven-dependency-plugin.version>
+    <maven-antrun-plugin.version>1.8</maven-antrun-plugin.version>
+    <maven.compiler.source>11</maven.compiler.source>
```

Comment:
Can we have code in 11 but target SE 8, for example?

Reply:
DAPR is a brand new project, and applications written for it should be targeting Java 11. Developers can reuse existing Java libraries and frameworks written for older versions of Java, for sure, but I see no point in having this SDK supporting Java 8.

Reply:
I agree that new Java projects should be in Java 11, mainly because it is the latest LTS version. My question here is, what should be the minimum version to allow existing Java apps to use Dapr? - not 100%, of course. It seems that prior to 11, the previous LTS version for Java is 8. I would like to hear from customers that want to use Dapr if Java 11 is OK for them now. We should eventually migrate to Java 11, my concern is if doing it now will hurt adoption.

https://www.oracle.com/technetwork/java/java-se-support-roadmap.html

Reply:
We may need to schedule a call. Hit me on brborges @ ms for later this week.

Reply:
I created an issue to discuss this with the broader community: https://github.com/dapr/java-sdk/issues/17